Sources/TokamakDemo/ShapeStyleDemo.swift

```diff
@@ -92,14 +92,44 @@ struct ShapeStyleDemo: View {
     HStack {
       VStack {
         Text("Primary")
-          .foregroundStyle(PrimaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.primary)
         Text("Secondary")
-          .foregroundStyle(SecondaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.secondary)
         Text("Tertiary")
-          .foregroundStyle(TertiaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.tertiary)
         Text("Quaternary")
-          .foregroundStyle(QuaternaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.quaternary)
       }
+      VStack {
+        Text("Primary")
+          .foregroundStyle(HierarchicalShapeStyle.primary)
+        Text("Secondary")
+          .foregroundStyle(HierarchicalShapeStyle.secondary)
+        Text("Tertiary")
+          .foregroundStyle(HierarchicalShapeStyle.tertiary)
+        Text("Quaternary")
+          .foregroundStyle(HierarchicalShapeStyle.quaternary)
+      }
+      .foregroundStyle(Color.red, Color.green, Color.blue)
+    }
+    VStack {
+      Rectangle()
+        .fill(
+          LinearGradient(
+            colors: [.red, .green, .blue],
+            startPoint: .bottomLeading,
+            endPoint: .topTrailing
+          )
+        )
+      Rectangle()
+        .fill(
+          RadialGradient(
+            colors: [.red, .green, .blue],
+            center: .topLeading,
+            startRadius: 50,
+            endRadius: 100
+          )
+        )
```

Comment:
Explicitly set frame to avoid layout issue on Safari.

```suggestion
      Rectangle()
        .fill(
          LinearGradient(
            colors: [.red, .green, .blue],
            startPoint: .bottomLeading,
            endPoint: .topTrailing
          )
        )
        .frame(width: 300, height: 150)
      Rectangle()
        .fill(
          RadialGradient(
            colors: [.red, .green, .blue],
            center: .topLeading,
            startRadius: 50,
            endRadius: 100
          )
        )
        .frame(width: 300, height: 150)
```